subprojects/common/elastic/src/main/kotlin/com/avito/android/elastic/HttpElasticClient.kt

```diff
@@ -60,9 +61,13 @@ internal class HttpElasticClient(
 
             params.putAll(metadata)
 
+            val formattedDate = isoDate.format(now)
+
+            dateFormatChecker.check(formattedDate)
```

Comment:
We've discussed feature toggles for new temporary fail-fast checks
up to you

Reply:
flag added